components/button/docs/tool.yml

```diff
@@ -0,0 +1,65 @@
+name: Tool
```

Comment:
deleting?

Reply:
Yup, but we're not removing anything here; this has to be backwards compatible.